### ISSUE TITLE
sso_session

### ISSUE THREAD

**Issue by acraftydev** [awslabs/amazon-ecs-local-container-endpoints#278]
if you using multiple sso profiles and use the sso_session config in .aws/config it currently fails to correctly use a shared sso session.

version 1.44.298 of the AWS go sdk introduced support for sso_session https://github.com/aws/aws-sdk-go/releases/tag/v1.44.298

There is already a pr to bump the aws sdk, https://github.com/awslabs/amazon-ecs-local-container-endpoints/pull/275

Is a new release planned or is there something i can help with to prepare a new release?

**Comment by evgenyslab** [comment]
Yes - We are seeing same problem - named sso profiles are NOT respected which is very problematic.

Work around now is to run `aws configure sso` without setting `SSO Session name` :(
